flang/test/Lower/do_concurrent_local_default_init.f90

```diff
@@ -0,0 +1,52 @@
+! Test default initialization of DO CONCURRENT LOCAL() entities.
+! RUN: bbc -emit-hlfir -I nowhere -o - %s | FileCheck %s
```

Comment:
Is `-I nowhere` needed?

Reply:
With bbc it hides the intrinsic modules, which prevents generation of runtime type info globals for derived type, making the test faster to run and a lot easier to debug if needed.